### PR TITLE
hover text (#20)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
- #20 hover text on all components
- support date & time as well as datetime
- allow lov as a list of date or datetime ...
- rename tp_varname as updateVarName
- rename tp_updatevars as updateVars
- #65 option to change responsive to False in plot_config property

### CODE REVIEW

File: gui/doc/shared.csv [hunk 1]
```diff
@@ -3,3 +3,4 @@ id,str,"""""",The identifier that will be assigned to the rendered HTML componen
 properties,"dict[str, any]",None,Bound to a dictionary that contains additonnal properties for this element.
 class_name,str,None,"CSS class name that will be associated with the generated HTML Element.
 This class names will be added to the default <code>taipy-&lt;element_type&gt;</code>."
+hover_text,dynamic(str),None,"Hover information provides tips on what a control does."
```
Comment (FabienLelaquais) [inline gui/doc/shared.csv:6]: Actually... the text could be anything...
What about:
"Information that is displayed when the user hovers over this element"
?

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me

File: taipy/gui/utils/date.py [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
-def date_to_ISO(date: datetime) -> str:
-    # return date.isoformat() + 'Z', if possible
-    try:
-        return date.astimezone(utc).isoformat()
-    except Exception:
-        # astimezone() fails on Windows for pre-epoch times
-        # See https://bugs.python.org/issue36759
-        warnings.warn("There is some problems with date parsing to ISO!")
-        return date.isoformat()
+def date_to_ISO(date_val: t.Union[datetime, date, time]) -> str:
```
Comment (FabienLelaquais) [inline taipy/gui/utils/date.py:8]: Shouldn't this be hidden from users?

**Review by FabienLelaquais — APPROVED** [review]
Great job.
As for the Chart thing... if it works, it works...

File: taipy/gui/utils/date.py (hunk 2 above)
Comment (FredLL-Avaiga) [reply taipy/gui/utils/date.py:8]: it will in the next PR

**Pull request merged**